websocket.go

```diff
@@ -220,18 +223,18 @@ func (c *Client) Send(dst *network.ServerIdentity, path string, buf []byte) ([]b
 		if err != nil {
 			return nil, NewClientError(err)
 		}
-		c.si = dst
+		c.connections[dest] = conn
 	}
 	defer func() {
 		if !c.keep {
-			c.Close()
+			c.Close(dest)
```

Comment:
I think there should be a `Close()` closing all connections in addition to `Close(dest)`. Users will most likely call `Close()` in any cases.
Indeed, `c.Close()` error is not handled: it probably should be ;)